Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Footer.css';
+
+
+function Footer() {
+    return (
+        <div className='footer'>
+            <a>
+                <img src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTbaEe1ndIr1RsiqmO6MphfGPBFaW8g6zFdkdmy5pmVZttv-JXs'></img>
```

Comment:
This was a very creative solution instead of using font awesome. Good job getting it working.